lib/miq_automation_engine/service_models/miq_ae_service_metric_zones.rb

```diff
@@ -0,0 +1,4 @@
+module MiqAeMethodService
+  class MiqAeServiceMetricZones < MiqAeServiceModelBase
```

Comment:
@kevensen Tests are failing on this class.  I see `metric_zone` referenced in the [miq_ae_service_container_replicator.rb](https://github.com/ManageIQ/manageiq/blob/fine/lib/miq_automation_engine/service_models/miq_ae_service_container_replicator.rb#L10), but I am not sure what the association is referring to.

Can you clarify what they are and if they are required in the service model?